corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -748,15 +748,10 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
 
         self.afterRender = function () {
             self.$picker = $('#' + self.entryId);
-            cloudcareUtils.initDateTimePicker(self.$picker, _.extend({
```

Comment:
This is inside `DateTimeEntryBase`, which is a base class for `DateEntry` and `TimeEntry` - there's a lot of complicated overriding because the initialization options differ significantly between the two, so here I split it up into two different initializations and moved all the options to the `utils` file that calls the library